app/models/service/containers.rb

```diff
@@ -1,6 +1,10 @@
 module Service
   class Containers
-    def self.start_container!(wizard_state)
+    def errors
```

Comment:
I added two validations to configuration step; we should do the same to other steps when possible (fail fast and all that). Remaining errors are probably more related to the service side of things, methinks.